.github/workflows/tests.yaml

```diff
@@ -33,6 +33,12 @@ jobs:
             dependencies: locked
             coverage-driver: xdebug
             symfony-version: '6.4.*'
+          - operating-system: ubuntu-latest
+            php-version: '8.4'
```

Comment:
Ideally, we would like to test against 8.5 too. I'd suggest to add symfony 8 in the `symfony-version` and add to exclude testing against php < 8.4

Reply:
@Kocal it is possible. GA supports `if` condition.

As an example you could take a look at the PR https://github.com/infection/infection/pull/1892 and  what we did there in order to introduce Symfony 7.0 support

Reply:
I mean, directly with the matrix, like I was able to do with `include` from the 1st commit. 

It should be fine now, I see https://github.com/infection/infection/actions/runs/19081901778/job/54512486280?pr=2551 install some Symfony 8 packages, but there is still PHPBench that force Symfony 7.x for `symfony/console` or `symfony/filesystem`.